Tests/FluentAssertions.Specs/Primitives/DateTimeOffsetAssertionSpecs.cs

```diff
@@ -2069,6 +2070,17 @@ public void When_asserting_subject_datetimeoffset_should_have_same_date_as_anoth
             act.Should().Throw<XunitException>().WithMessage(
                 "Expected the date part of subject to be <2009-12-30>, but it was <2009-12-31>.");
         }
+
+        [Fact]
+        public void Successive_and_call_works()
```

Comment:
```suggestion
        public void Can_chain_follow_up_assertions()
```

Since we don't want to include the names of code elements in test names.

Same comment applies to the other new tests.